src/solver/optimisation/include/antares/solver/optimisation/LegacyFiller.h

```diff
@@ -10,7 +10,7 @@ namespace Antares::Optimization
 class LegacyFiller: public Optimisation::LinearProblemApi::LinearProblemFiller
 {
 public:
-    explicit LegacyFiller(const PROBLEME_HEBDO* problemeHebdo);
+    explicit LegacyFiller(const PROBLEME_HEBDO* problemeHebdo, bool forceNamingProblem = false);
```

Comment:
```suggestion
    explicit LegacyFiller(const PROBLEME_HEBDO* problemeHebdo, bool nameProblems = false);
```
one source of information from the way up makes the code more readable IMO